src/stories/link-filters/LinkFilters.tsx

```diff
@@ -1,51 +1,33 @@
 import { Links } from "../links/Links";
 
-export const LinkFilters = () => {
+export type LinkFilter = {
+  title: string;
+  href?: string;
+  counter: string;
+};
+
+export type LinkFiltersProps = {
+  filters: Array<LinkFilter>;
+};
+
+export const LinkFilters = (props: LinkFiltersProps) => {
   return (
     <div className="link-filters-container">
-      {list.map((item) => (
-        <div className="link-filters-tag-wrapper">
-          <Links
-            href={item.href}
-            linkText={item.title}
-            classNames="link-filters-tag"
-          />
+      {props.filters.map((item, index) => (
+        <div key={index} className="link-filters-tag-wrapper">
+          {item.href ? (
```

Comment:
Could this be an (un)ordered list with list items?

Reply:
Yup. I have changed it.